domain-models-runtime/src/test/java/org/folio/rest/tools/utils/ModuleName.java

```diff
@@ -0,0 +1,10 @@
+package org.folio.rest.tools.utils;
+
+public class ModuleName {
+  /**
+   * The module name with minus replaced by underscore, for example {@code mod_foo_bar}.
+   */
+  public static String getModuleName() {
```

Comment:
I don't like that this class and method is bundled with the runtime.. Can't this just be in test?

Reply:
For me `domain-models-runtime/src/test` is in test only.
Can you elaborate?

Reply:
Brain damage on my side.. Excellent.

Reply:
ModuleName is not in the fat jar:
```
unzip -l domain-models-runtime/target/domain-models-runtime-fat.jar|grep -i ModuleName
```